pymc3/tuning/starting.py

```diff
@@ -30,21 +34,21 @@ def find_MAP(start=None, vars=None, fmin=None,
     start : `dict` of parameter values (Defaults to `model.test_point`)
     vars : list
         List of variables to set to MAP point (Defaults to all continuous).
-    fmin : function
-        Optimization algorithm (Defaults to `scipy.optimize.fmin_bfgs` unless
```

Comment:
I'm happy to re-work this PR either choice on the breakage, but there wasn't really a decision made in #2468...